modules/store.js

```diff
@@ -0,0 +1,32 @@
+class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+
+    return books;
+  }
+
+  static addBook(book) {
```

Comment:
- [ ] I am currently unable to add an item. Kindly check your add book function 👍🏼

Reply:
@vickymarz Hey Vicky, could you check again? because the project seems to work perfectly okay, and you can add a new book without any problem. You need to click on the list tab to see all the added books